cloud.foundry.cli/build.gradle

```diff
@@ -68,9 +68,11 @@ jar {
     manifest {
         attributes 'Main-Class': 'cloud.foundry.cli.getservice.GetController',
                 'Class-Path': configurations.runtime.files.collect { "lib/$it.name" }.join(' ')
-        }
+    }
 
-        from {
-            configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
-        }
+    from {
+        configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
     }
+
+    baseName 'cf-control'
```

Comment:
Neat. That's probably the easiest way to control the JAR output filename.

I think in the future we might also just add a version number, if we get to a point where this becomes necessary.